setup.cfg

```
@@ -128,7 +128,7 @@ tests =
     # Type annotation stubs
     # http://mypy-lang.blogspot.com/2021/05/the-upcoming-switch-to-modular-typeshed.html
     types-Jinja2>=0.1.3
-    types-protobuf>=0.1.10
+    types-protobuf>=0.1.10,<5.29.1.20250402
```

This is surely a broken release https://github.com/python/typeshed/issues/13770 
```suggestion
    types-protobuf>=0.1.10,!=<5.29.1.20250402
```